schema.graphql

```diff
@@ -109,6 +129,11 @@ type Order{
 	items: [Item!]!
 	totalPrice: Int!
 	time: String!
+    customerId: String!
+    customerName: String!
+	arrivedTime: String!   #
```

Comment:
what's the meaning of # here? forgot to delete?

Reply:
I will delete it.